src/tools/logics.js

```diff
@@ -95,17 +95,23 @@ const testWord = (word) => {
 };
 
 // Funktio joka yhdistää Caesar-käännöksen ja testaa lauseen sanat sääntöjä vasten
-const uncipherSentences = (data) => {
```

Comment:
When you have domain / problem specific function name, like this one (uncipherSentences) it makes sense to give meaning to the input as well. 'data' does not tell much but 'sentences' are very easy for someone else to read and understand what is going on.